airflow/example_dags/plugins/workday.py

```diff
@@ -78,6 +78,9 @@ def next_dagrun_info(
 
     # [END howto_timetable_next_dagrun_info]
 
+    def validate(self):
```

Comment:
should it be separated ? seems like "real" bugfix which might be hidden in big mypy change ?

Reply:
Scrap that. I know. comment below.